spec/menu_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "menu"
+
+describe Menu do
+
+  let (:menu_items) {double :menu_items}
```

Comment:
What is this double being used in?

Reply:
I thought I needed it but I still don't grasp it well enough to know when it comes in useful so wasn't sure if I should take it out :'(